### ISSUE TITLE
Windows Installer ending up with Error: Error getting latest GitHub tag from GitHub exiting

### ISSUE THREAD

**Issue by shahidsaeed** [kercre123/WirePod#20]
I am using the latest release WirePodInstaller-v1.2.10.exe, it was working fine but all of the sudden the web interface stopped working, I tried to quit the app and re-launch multiple times, it starts the web server but the web interface doesn't load. After many tries I uninstalled the app, download it again and trying to install now but it is installer ending up with error: "Error getting latest GitHub tag from GitHub exiting: Get. I have attached the screenshot for your reference, I am using windows 10 64bit.

![Screenshot (1270)](https://github.com/user-attachments/assets/a2caa2a9-e7fc-4624-a070-ff383ed6c08e)

**Comment by shahidsaeed** [comment]
Thank you for release new version, with the new version 1.2.11.exe, I was able to install the software again and wirepod working now. I was able to connect my vector too, but unfortunately the voice commands not working at all, voice commands were blocked by the DDL last update and I didn't subscribed for their life time or monthly subscription to activate it. Since then Vector doesn't respond to any voice command. with wire-pod I can interact with vector and do all basic stuff. I hope there will be a workaround to activate the voice commands as well.

Thank you for your great work so far.

**Comment by kercre123** [comment]
DDL hasn't released an update for years. Their servers have also been down for more than a year by this point. Voice command blocking was done server-side, not robot-side. So, since wire-pod acts as a whole new server for Vector, your subscription status doesn't matter.

Were you able to activate just fine? Maybe wire-pod's speech-to-text engine isn't understanding you? You can check the logs to see what it is transcribing your text to (web interface -> Logs). Could you also check "See debug logs" and copy-paste them here?

**Comment by shahidsaeed** [comment]
Yes, the robot is activated, I also tried to change the firmware between 1.8 and 2.0.1 (vise-versa), but I couldn't activate the voice commands. For example when I say Hey Vector it doesn't respond to anything. When I press the button it respond but then there is some error in the log file. I am putting the log data below for your reference. When I type text in control section on wire-pod. it speaks the text, when it see me it speaks my name but when I say hey vector, it doesn't respond anything. The only way to make him respond is to press his button but that also doesn't work due to some EOF error.

LOG DATA:

2024.08.03 04:37:10: Commit SHA: 63f9d28
2024.08.03 04:37:10: Initializing variables
2024.08.03 04:37:10: This version of wire-pod is packaged. Set vars to include UserConfigDir...
2024.08.03 04:37:10: SDK info path: C:\Users\Shahid Saeed/.anki_vector/
2024.08.03 04:37:10: API config JSON created
2024.08.03 04:37:10: open C:\Users\Shahid Saeed\AppData\Roaming\wire-pod\vosk\models: The system cannot find the path specified.
2024.08.03 04:37:10: Reading session certs for robot IDs
2024.08.03 04:37:10: Initiating vosk voice processor with language 
2024.08.03 04:37:10: Loading plugins
2024.08.03 04:37:10: Unable to load plugins:
2024.08.03 04:37:10: open ./plugins: The system cannot find the file specified.
2024.08.03 04:37:10: Wire-pod is not setup. Use the webserver at port 8080 to set up wire-pod.
2024.08.03 04:37:10: Starting SDK app
2024.08.03 04:37:10: [1;36mConfiguration page: http://192.168.1.150:8080[0m
2024.08.03 04:37:10: BLE API is unregistered
2024.08.03 04:37:44: Downloading https://github.com/kercre123/vosk-models/raw/main/vosk-model-small-en-us-0.15.zip to C:\Users\SHAHID~1\AppData\Local\Temp/vosk-model-small-en-us-0.15.zip
2024.08.03 04:37:53: Configuration changed, writing to disk
2024.08.03 04:37:53: Opening VOSK model (C:\Users\Shahid Saeed\AppData\Roaming\wire-pod\vosk\models\en-US\model)
2024.08.03 04:37:54: Initializing VOSK recognizers
2024.08.03 04:37:54: VOSK initiated successfully
2024.08.03 04:37:54: Running recognizer test
2024.08.03 04:37:54: Using general recognizer
2024.08.03 04:37:55: Text (from test):how are you
2024.08.03 04:37:55: Vosk test successful! (Took 937.7727ms)
2024.08.03 04:37:55: Reloaded voice processor successfully
2024.08.03 04:37:55: Configuration changed, writing to disk
2024.08.03 04:37:55: Registering escapepod.local on network (loop)
2024.08.03 04:37:55: Initiating TLS listener, cmux, gRPC handler, and REST handler
2024.08.03 04:37:55: Starting chipper server at port 443
2024.08.03 04:37:55: Starting chipper server at port 8084 for 2.0.1 compatibility
2024.08.03 04:38:07: Vector discovered on network, broadcasting mDNS
2024.08.03 04:38:08: Broadcasting mDNS now (outside of timer loop)
2024.08.03 04:58:10: Token: Incoming Associate Primary User request
2024.08.03 04:58:10: Current time: 2024-08-03T04:58:10.811588+04:00
2024.08.03 04:58:10: Token expires: 2024-09-03T04:58:10.811588+04:00
2024.08.03 04:58:10: ESN not found in store or this is an associate primary user request, act as if this is a new robot
2024.08.03 04:58:10: Adding 192.168.1.149 to TokenHashStore
2024.08.03 04:58:10: UUID for this token request: f35876d3-7122-4871-b85c-6adc9153e10a
2024.08.03 04:58:11: Jdocs: Incoming ReadDocs request, Robot ID: vic:00a158a6, Item(s) to return: 
2024.08.03 04:58:11: [doc_name:"vic.AppTokens"]
2024.08.03 04:58:11: Adding 00a158a6 to bot info store
2024.08.03 04:58:11: App tokens jdoc not found for this bot, trying bots in TokenHashStore
2024.08.03 04:58:11: GUID and hash successfully written for 00a158a6
2024.08.03 04:58:11: ReadJdocs: bot 00a158a6 matched with IP 192.168.1.149 in token store
2024.08.03 04:58:11: Removing 192.168.1.149 from temporary token-hash store
2024.08.03 04:58:11: Outputting session cert to C:\Users\Shahid Saeed\.anki_vector\Vector-B9R9-00a158a6.cert
2024.08.03 04:58:11: WriteToIniPrimary: bot already in INI matched, setting info
2024.08.03 04:58:11: WriteToIniPrimary: successfully wrote INI
2024.08.03 04:58:11: Removing 192.168.1.149:38981 from cert-write store
2024.08.03 04:58:11: Session certificate successfully output
2024.08.03 04:58:11: Jdocs: Incoming ReadDocs request, Robot ID: vic:00a158a6, Item(s) to return: 
2024.08.03 04:58:11: [doc_name:"vic.RobotSettings" doc_name:"vic.RobotLifetimeStats" doc_name:"vic.AccountSettings" doc_name:"vic.UserEntitlements"]
2024.08.03 04:58:11: Jdocs: Incoming WriteDoc request, Item to write: vic.RobotSettings, Robot ID: vic:00a158a6
2024.08.03 04:58:12: Jdocs: Incoming WriteDoc request, Item to write: vic.RobotLifetimeStats, Robot ID: vic:00a158a6
2024.08.03 04:58:12: Jdocs: Incoming WriteDoc request, Item to write: vic.AccountSettings, Robot ID: vic:00a158a6
2024.08.03 04:58:12: Jdocs: Incoming WriteDoc request, Item to write: vic.UserEntitlements, Robot ID: vic:00a158a6
2024.08.03 04:58:15: Successfully got jdocs from 00a158a6
2024.08.03 04:58:28: Jdocs: Incoming WriteDoc request, Item to write: vic.AccountSettings, Robot ID: vic:00a158a6
2024.08.03 04:58:28: Jdocs: Incoming WriteDoc request, Item to write: vic.RobotSettings, Robot ID: vic:00a158a6
2024.08.03 05:00:59: Configuration changed, writing to disk
2024.08.03 05:06:49: Configuration changed, writing to disk
2024.08.03 05:07:15: Connecting to 00a158a6 with GUID C5l12tAhK4t7IbGNoXoCbg==
2024.08.03 05:07:16: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:07:19: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:07:25: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:07:29: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:07:45: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:07:45: Jdocs: Incoming WriteDoc request, Item to write: vic.RobotSettings, Robot ID: vic:00a158a6
2024.08.03 05:07:54: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:07:58: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:08:03: Updating vic.RobotSettings (source: sdkapp)
2024.08.03 05:10:05: Bot 00a158a6 Stream type: OPUS
2024.08.03 05:10:05: (Bot 00a158a6, Vosk) Processing...
2024.08.03 05:10:05: Using general recognizer
2024.08.03 05:10:11: EOF
2024.08.03 05:10:11: Bot 00a158a6 Intent Sent: intent_system_noaudio
2024.08.03 05:10:11: Bot 00a158a6 Parameters Sent:map[error:EOF]
2024.08.03 05:15:25: Closing SDK connection for 00a158a6, source: connTimer
